src/core/public/index.scss

```diff
@@ -7,5 +7,54 @@
 // Mixins provide generic code expansion through helpers
 @import '@elastic/eui/src/global_styling/mixins/index';
 
+$kbnGlobalNavClosedWidth: 53px;
+$kbnGlobalNavOpenWidth: 180px;
+$kbnGlobalNavLogoHeight: 70px;
+$kbnGlobalNavAppIconHeight: $euiSizeXXL + $euiSizeXS;
+
+#kibana-body {
```

Comment:
We also really try to avoid putting selectors directly in the `index` manifest files. It was separated before, is there a reason it must be included in this file?